### PR TITLE
DX-2470 add modifyCallBxml Functionality

### PR DESCRIPTION
**Pull request by ajrice6713**

### CODE REVIEW

File: Bandwidth.Standard/Voice/Bxml/Bxml.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+namespace Bandwidth.Standard.Voice.Bxml
+{
+  /// <summary>
+  ///   Bxml class for Bandwidth XML
+  /// </summary>
+  [XmlRoot("Bxml", Namespace = "")]
+  public class BXML : IXmlSerializable
+  {
+    private static readonly XmlSerializer Serializer = new XmlSerializer(typeof (BXML), "");
+    private readonly List<IVerb> _list = new List<IVerb>();
```
Comment (hamermike) [inline Bandwidth.Standard/Voice/Bxml/Bxml.cs:19]: Indent a smidge.

File: Bandwidth.Standard/Voice/Bxml/Bxml.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+        private static readonly Regex XML_REGEX = new Regex("&lt;([a-zA-Z//].*?)&gt;");
+
+        private static readonly Regex SPEAK_SENTENCE_REGEX = new Regex("<SpeakSentence.*?>.*?<\\/SpeakSentence>");
```
Comment (hamermike) [inline Bandwidth.Standard/Voice/Bxml/Bxml.cs:23]: CRAZY_CAPS on these. Private fields should be camel casing and prefixed with `_` as you did with `_list`.

File: Bandwidth.Standard/Voice/Bxml/Bxml.cs [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    public BXML(params IVerb[] verbs)
+    {
+      _list.AddRange(verbs);
+    }
+
+    XmlSchema IXmlSerializable.GetSchema() => null;
+
+    void IXmlSerializable.ReadXml(XmlReader reader)
+    {
+      throw new NotImplementedException();
```
Comment (hamermike) [inline Bandwidth.Standard/Voice/Bxml/Bxml.cs:45]: This throws an exception when called. This should be implemented before we release.

File: Bandwidth.Standard/Voice/Bxml/Bxml.cs [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    void IXmlSerializable.WriteXml(XmlWriter writer)
+    {
+      var ns = new XmlSerializerNamespaces();
+      ns.Add("", "");
+      foreach (var verb in _list)
+      {
+        var serializer = new XmlSerializer(verb.GetType(), "");
+        serializer.Serialize(writer, verb, ns);
+      }
+    }
```
Comment (hamermike) [inline Bandwidth.Standard/Voice/Bxml/Bxml.cs:57]: This doesn't appear to do anything at the moment. Currently this serializes but does nothing with it and will be lost once the scope is outside the function.

File: Bandwidth.Standard/Voice/Bxml/Bxml.cs (hunk 3 above)
Comment (ajrice6713) [reply Bandwidth.Standard/Voice/Bxml/Bxml.cs:45]: @hamermike this comes from Response.cs - can we just remove it? Not even sure what this method is supposed to do

File: Bandwidth.Standard/Voice/Bxml/Bxml.cs (hunk 4 above)
Comment (ajrice6713) [reply Bandwidth.Standard/Voice/Bxml/Bxml.cs:57]: Hmm - without this I get 

```
/Users/***/csharp-sdk/Bandwidth.Standard/Voice/Bxml/Bxml.cs(16,23): error CS0535: 'BXML' does not implement interface member 'IXmlSerializable.WriteXml(XmlWriter)' [/Users/***/csharp-sdk/Bandwidth.Standard/Bandwidth.Standard.csproj]
```

**Comment by brianluisgomez** [comment]
Tested the function and it does work correctly and update calls.

**Review by brianluisgomez — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -86,6 +86,24 @@ response.ToBXML();
...
+
+var speakSentence = new SpeakSentence();
+speakSentence.Sentence = "This text is new!";
+speakSentence.Voice = "susan";
+speakSentence.Gender = "female";
+speakSentence.Locale = "en_US";
+
+var bxml = new BXML(speakSentence);
+
+await client.Voice.APIController.ModifyCallBxml("1111111", callId, bxml.ToString());
```
Comment (hamermike) [inline README.md:103]:
```suggestion
await client.Voice.APIController.ModifyCallBxml("1111111", callId, bxml.ToBXML());
```

**Review by hamermike — CHANGES_REQUESTED** [review]

**Pull request merged**